Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -109,6 +119,8 @@ ErrorOr<void> PNGImageDecoderPlugin::initialize()
         *read_data = read_data->slice(length);
     });
 
+    png_set_error_fn(png_ptr, nullptr, log_png_error, log_png_warning);
```

Comment:
> The replacement functions must do a longjmp to the last setjmp location if setjmp/longjmp method of error handling is used. If error_fn or warning_fn is NULL, the default functions for error handling shall be used.

IIRC we do use setjmp/longjmp for errors.

Reply:
Added a call to `png_longjmp(png_ptr, 1);` in the custom error log function.
I don't believe this is required for the warning function.